lib/rubocop/cop/layout/class_structure.rb

```diff
@@ -23,6 +23,10 @@ module Layout
       # * Private instance methods
       #
       # You can configure the following order:
+      # NOTE: Simply enabling the cop with `Enabled: true` will not use
```

Comment:
I think this note should probably be after the example or before the previous line.

Reply:
When generating the site from the doc, the example appears last, so it would be better to place this before the example.

Reply:
I moved this note before the example.